tests/templates/kuttl/keycloak-user-info/03-setup-keycloak.yaml.j2

```diff
@@ -0,0 +1,140 @@
+                - name: USER_INFO_FETCHER_CLIENT_SECRET
+                  valueFrom:
+                    secretKeyRef:
+                      name: user-info-fetcher-client-credentials
+                      key: clientSecret
+              command:
+                - bash
+                - -x
+                - -euo
+                - pipefail
+                - -c
+                - |
+                  COMMON_FLAGS="--config kcadm.conf --truststore /tls/truststore.p12 --trustpass changeit"
+
+                  echo "Configuring credentials"
+                  kcadm.sh config credentials \$COMMON_FLAGS --server https://keycloak.$NAMESPACE.svc.cluster.local:8443 --realm master --user admin --password "\$KEYCLOAK_ADMIN_PASSWORD"
+
+                  echo "Adding users"
+                  ALICE_UUID=\$(kcadm.sh create users \$COMMON_FLAGS -s username=alice -s firstName=Alice -s lastName=Adams -s email=alice@mycorp.org -s enabled=true -o | jq -r .id)
+                  BOB_UUID=\$(  kcadm.sh create users \$COMMON_FLAGS -s username=bob   -s firstName=Bob   -s lastName=Bush  -s email=bob@mycorp.org   -s enabled=true -o | jq -r .id)
+
+                  echo "Setting user passwords"
+                  kcadm.sh set-password \$COMMON_FLAGS --username alice --new-password "\$ALICE_PASSWORD"
+                  kcadm.sh set-password \$COMMON_FLAGS --username bob   --new-password "\$BOB_PASSWORD"
+
+                  echo "Creating the superset-admin group"
+                  SUPERSET_ADMIN_GROUP_UUID=\$(kcadm.sh create groups \$COMMON_FLAGS -s name=superset-admin -o | jq -r .id)
+                  echo "Adding alice to the superset-admin group"
+                  kcadm.sh update \$COMMON_FLAGS "users/\$ALICE_UUID/groups/\$SUPERSET_ADMIN_GROUP_UUID"
+
+                  echo "Creating user-info-fetcher client"
+                  kcadm.sh create clients \$COMMON_FLAGS -f - << EOF || true
+                  {
+                    "clientId": "user-info-fetcher",
+                    "enabled": true,
+                    "protocol": "openid-connect",
+                    "clientAuthenticatorType": "client-secret",
+                    "secret": "\$USER_INFO_FETCHER_CLIENT_SECRET",
+                    "redirectUris": [
+                      "*"
+                    ],
+                    "webOrigins": [
+                      "*"
+                    ],
+                    "publicClient": false,
+                    "standardFlowEnabled": false,
+                    "implicitFlowEnabled": false,
+                    "directAccessGrantsEnabled": false,
+                    "serviceAccountsEnabled": true,
+                    "attributes": {
+                      "oidc.ciba.grant.enabled": "true",
+                      "oauth2.device.authorization.grant.enabled": false
+                    }
+                  }
+                  EOF
+
+                  kcadm.sh add-roles --uusername service-account-user-info-fetcher --rolename admin \$COMMON_FLAGS
```

Comment:
`--username`? Or is this a weird keycloakism?

Reply:
It's an 'ism :/